lua/core/paramset.lua

```diff
@@ -445,8 +448,12 @@ function ParamSet:write(filename, name)
       end
     end
     io.close(fd)
-    if self.action_write ~= nil then 
-      self.action_write(filename,name)
+    if self.action_write ~= nil then
+      if pset_number ~= nil then
```

Comment:
i don't think you need to check for nil: just sending nil is fine

(just simplifies the code, not functional)